assets/js/base/components/filter-submit-button/index.tsx

```diff
@@ -33,6 +35,7 @@ const FilterSubmitButton = ( {
 				'wp-block-button__link',
 				'wc-block-filter-submit-button',
 				'wc-block-components-filter-submit-button',
+				{ 'show-loading-state': isLoading },
```

Comment:
Nit: we usually use `is-loading` as the class for loading components.

Reply:
Fixed 👌